### PR TITLE
Mu 12

### PR DESCRIPTION
**Pull request by Staroz**
Please, Review this PR, these are Tests in Trello with Playwright.

### CODE REVIEW

File: playwright/.github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Playwright Tests
```
Comment (sphinix27) [inline playwright/.github/workflows/playwright.yml]: Why this file is a duplicated of the previous? Remove one of them, only one should be required.

File: playwright/tests/testTrello.spec.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        await page.locator('[data-testid="create-board-tile"]').click();
+        await page.locator('[data-testid="create-board-title-input"]').click();
+        await page.getByTestId('create-board-title-input').fill(credentials.boardName);
+        await page.getByTestId('create-board-submit-button').click();
+        await page.locator('[data-testid="board-name-display"]').click();
+        await page.locator('[data-testid="board-name-input"]').fill(credentials.newBoardName);
+        await page.locator('[data-testid="board-name-input"]').press('Enter');
+        await expect(page.locator('[data-testid="board-name-display"]')).toContainText(credentials.newBoardName);
+    })
+});
```
Comment (sphinix27) [inline playwright/tests/testTrello.spec.js:60]: Looks good, remember to add a test or something to delete that board.

**Review by sphinix27 — CHANGES_REQUESTED** [review]
Add the changes and we can merge this

File: playwright/tests/testTrello.spec.js (hunk 2 above)
Comment (Staroz) [reply playwright/tests/testTrello.spec.js:60]: in line 28, I create an afterEach that deletes the board.

**Comment by sphinix27** [comment]
Pipeline is failing, it shouldn't

**Review by sphinix27 — CHANGES_REQUESTED** [review]
Fix the pipeline for playwright,  at least 3 runs should pass

**Comment by Staroz** [comment]
Hi, I've seen for convenience closed this PR, and fix to all observations in "Branch mu_013", I hope your answer, thanks.

**Review by sphinix27 — APPROVED** [review]
lgtm

**Pull request merged**